Subscriber/CheckoutSubscriber.php

```diff
@@ -443,9 +442,9 @@ private function shouldRedirectToStep2(Shopware_Controllers_Frontend_Checkout $s
             return true;
         }
 
-        $paymentMethod = $adyenPaymentMethods->fetchByTypeOrId($selectedType);
         if (!$paymentMethod->getValue('details') && !$paymentMethod->isStoredPayment()) {
-            $subject->View()->assign('sAdyenSetSession', json_encode($paymentMethod->getRawData()));
```

Comment:
Seems this data was always set to incorrect data for Checkout API, 
updated it to contain correct data when it was not handled by Web components.